newsletters.md

```diff
@@ -27,3 +27,5 @@ version: 1
   {%- endif -%}
   {%- endfor -%}
 </ul>
+
+{% include newsletter-signup.html %}
```

Comment:
Suggest moving this to the top of the page, probably right after the end of the YAML header (line 11) so it displays  below the Newsletters page heading but above the list.  The reason is that the list of newsletters is going to get longer over time, so this box will move further down the page few people ever see it.

You may also wish to add it to each newsletter (in the same position right below the end of the YAML header), as those pages may receive a lot of incoming link traffic.  There are a few ways to do this, such as creating a layout special to newsletters, putting it in the page layout triggered by a `news_signup: true` option in the YAML header, or just putting the `{% include %}` in each newsletter---the later is the easiest for now, so I'd suggest that. 

Otherwise this LGTM.  I tested and it took me to the mailchimp page and from there back to the live site, but they didn't ask me for double opt-in, so maybe it's in some sort of test mode?